<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PixmapExportDialog</class>
 <widget class="QDialog" name="PixmapExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export as portable anymap</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="mapTypeLabel">
       <property name="text">
        <string>Map type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" rowspan="2">
      <widget class="QRadioButton" name="grayscaleBinRadio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Grayscale</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QRadioButton" name="colorBinRadio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2" colspan="2">
      <widget class="QRadioButton" name="grayscaleTxtRadio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Grayscale</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2" rowspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="binLabel">
       <property name="text">
        <string>Binary</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="colorTxtRadio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="overrideCheckbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Override brightness</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSpinBox" name="overrideValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="exportButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exportButton</sender>
   <signal>accepted()</signal>
   <receiver>PixmapExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton</sender>
   <signal>rejected()</signal>
   <receiver>PixmapExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overrideCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>overrideValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
